nova/virt/vmwareapi/vm_util.py

```diff
@@ -2028,3 +2041,21 @@ def rename_vm(session, vm_ref, instance):
 def is_vim_instance(o, vim_type_name):
     return isinstance(o, sudsobject.Factory.subclass(vim_type_name,
                                                      sudsobject.Object))
+
+
+def vm_device_change(session, vm_ref, device_change):
+    """Sets the deviceChange and invokes a ReconfigVM_Task.
+
```

Comment:
I think that's too technical. We can see this in the code. I'd go with something more prose like `Reconfigure a VM to add/replace/remove devices`